src/php/Config/PhelConfig.php

```diff
@@ -0,0 +1,143 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Phel\Config;
+
+use JsonSerializable;
+use Phel\Build\BuildConfig;
+use Phel\Command\CommandConfig;
+use Phel\Filesystem\FilesystemConfig;
+use Phel\Formatter\FormatterConfig;
+use Phel\Interop\InteropConfig;
+
+final class PhelConfig implements JsonSerializable
+{
+    private array $srcDirs = ['src/phel'];
+    private array $testDirs = ['tests/phel'];
```

Comment:
I would be nice if you can add type annotations for all arrays, like `list<string>`

Reply:
Done.